README.md

```diff
@@ -1,55 +1,21 @@
-# db-scripts
```

Comment:
@bobg - We should add a line to the README instructing the reader to configure MySQL with a password. (1) The setupdb.py script fails if the password for the database is blank. I'm sure we could recode the script to work, but it seems like the database should be password protected anyway.